### PR TITLE
[CORE-10501] Prepare for published contentful content with validation errors

### PR DESCRIPTION
**Pull request by eliasbraun**
- https://jira.nebenan.de/browse/CORE-10501

### CODE REVIEW

File: src/contentful/utils.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ export const getContentfulRequest = (type, contentQuery) => {
 export const formatImage = (image, assets) => {
   const ref = assets[image.sys.id];
   if (!ref) return null;
-  return `https:${ref.fields.file.url}`;
+  return `https:${ref.fields.file?.url}`;
```
Comment (peterhass) [inline src/contentful/utils.js]: would rather return `null` instead of `https://` in error case

**Review by peterhass — CHANGES_REQUESTED** [review]

File: src/contentful/utils.js (hunk 1 above)
Comment (eliasbraun) [reply src/contentful/utils.js]: Would you go for this instead? 
```
// ... 
if (!ref.fields.file?.url) return null;
return `https:${ref.fields.file.url}`;
// ...
```
Also: would you still insert a comment here?

Comment (peterhass) [reply src/contentful/utils.js]: Comment: Yes
```js
const ref = assets[image.sys.id];
const url = ref?.fields?.file?.url;
if (!url) return null;

return `https:${url}`;
```

**Comment by eliasbraun** [comment]
Closed due to inactivity + already solved in core contentful PR https://github.com/good-hood-gmbh/nebenan-frontend/pull/1763

**Pull request closed without merging**